rest/rest-smartproxy/src/main/java/org/ow2/proactive_grid_cloud_portal/smartproxy/RestSmartProxyImpl.java

```diff
@@ -86,6 +82,7 @@
 
 import com.google.common.base.Throwables;
 import com.google.common.collect.Lists;
+import org.ow2.proactive_grid_cloud_portal.common.SortSpecifierRestContainer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.ow2.proactive_grid_cloud_portal.common.SortSpecifierRestContainer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)